tools/emscripten.meson.cross

```diff
@@ -0,0 +1,14 @@
+[binaries]
+c = 'emcc'
+cpp = 'em++'
+ar = 'emar'
+fortran = '/usr/bin/gfortran'
```

Comment:
It doesn't work if you just pass `gfortran`? 

Also, scikit-image doesn't have any Fortran as far as I'm aware. So maybe leave it commented for now?

Reply:
Not important, OK to keep it as is.